README.adoc

```diff
@@ -114,22 +114,8 @@ classes that reduce the need to write custom code. Instead, you can focus on the
 logic.
 
 To configure your job, you must first create a Spring `@Configuration` class like the following example in
-`src/main/java/com/exampe/batchprocessing/BatchConfiguration.java`:
-
-====
-[source,java,tabsize=2]
-----
-include::complete/src/main/java/com/example/batchprocessing/BatchConfiguration.java[tag=setup]
-
-    ...
-
-}
-----
-====
-
-For starters, the `@EnableBatchProcessing` annotation adds many critical beans that
-support jobs and save you a lot of leg work. This example uses a memory-based database
-(provided by `@EnableBatchProcessing`), meaning that, when it is done, the data is gone. It also autowires a couple factories needed further below.
+`src/main/java/com/exampe/batchprocessing/BatchConfiguration.java`. This example uses a memory-based database,
```

Comment:
I would make the new sentence be a note. I would also reword it into two sentences, so that it's not such a complex sentence:

This example uses an in-memory database. When the application shuts down, no data is retained.